app/src/main/java/de/berlindroid/zepatch/ui/PatchableToReducedBitmap.kt

```diff
@@ -39,32 +35,61 @@ fun PatchableToReducedBitmap(
     computeReducedBitmap: () -> Unit = {},
     onColorCountChanged: (Int) -> Unit = {}
 ) {
-    Column(modifier = modifier.fillMaxWidth()) {
+    // Local text state so users can clear or type partial numbers without snapping back to default
+    var colorText by rememberSaveable { mutableStateOf(colorCount.toString()) }
+
+    // Keep local text in sync if colorCount changes from outside (e.g., recomputations)
+    LaunchedEffect(colorCount) {
+        colorText = colorCount.toString()
+    }
+
+    val focusManager = LocalFocusManager.current
+
+    Column(
+        modifier = modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        WizardSectionTitle(
+            title = "Reduce Colors",
+            helpText = "Choose the number of colors and generate a simplified bitmap suitable for stitching."
```

Comment:
Nitpick: Is it stitching or embroidering. I'm voting for replacing `stitching` with `embroidering`.